### PR TITLE
Verified the use of "Personal Data", expand list of module considerations

### PR DESCRIPTION
**Pull request by jwfuller**
- Verified the use of "Personal Data (PD)" instead of "Personally Identifiable Information (PII)". Personal Data is a broader term that encompasses Personally Identifiable Information. Organizations that are subject to GDPR are obligated to protect a user's personal data.
- Expand the modules specific checklist to include processing and logging of Personal Data.
- Organize list of types of personal data to emphasize data that is identifiable by itself.

### CODE REVIEW

**Comment by inkuss** [comment]
I approve the changes.

File: PERSONAL_DATA_DISCLOSURE.md [hunk 1]
```diff
@@ -1,58 +1,81 @@
...
+* [General Data Protection Regulation (GDPR)](https://eur-lex.europa.eu/legal-content/EN/TXT/HTML/?uri=CELEX:02016R0679-20160504)
+  * [What are identifiers and related factors? (ico.org.uk)](https://ico.org.uk/for-organisations/guide-to-data-protection/guide-to-the-general-data-protection-regulation-gdpr/what-is-personal-data/what-are-identifiers-and-related-factors/)
+  * [What is the meaning of 'relates to'? (ico.org.uk)](https://ico.org.uk/for-organisations/guide-to-data-protection/guide-to-the-general-data-protection-regulation-gdpr/what-is-personal-data/what-is-the-meaning-of-relates-to/)
+  * [Opinion 4/2007 on the concept of personal data (Article 29 working party)](https://ec.europa.eu/justice/article-29/documentation/opinion-recommendation/files/2007/wp136_en.pdf)
 * [California Consumer Privacy Act (CCPA)](https://oag.ca.gov/privacy/ccpa)
-* [U.S. Department of Labor: Guidance on the Protection of Personal Identifiable Information](https://www.dol.gov/general/ppii)
+* [U.S. Department of Labor: Guidance on the Protection of Personal Identifiable Information](https://www.dol.gov/general/pPD)
```
Comment (dltj) [inline PERSONAL_DATA_DISCLOSURE.md]: Global search-and-replace gone awry here.

File: PERSONAL_DATA_DISCLOSURE.md [hunk 2]
```diff
@@ -1,58 +1,81 @@
-
-
 ## Overview
-The purpose of this form is to disclose the types of personal data stored by each module.  This information enables those hosting FOLIO to better manage and comply with various privacy laws and restrictions, e.g. GDPR.
 
-It's important to note that personal data is not limited to that which can be used to identify a person on it's own (e.g. Social security number), but also data used in conjunction with other data to identify a person (e.g. date of birth + city + gender).
+The purpose of this form is to disclose the types of personal data[^1] (PD) stored by each module.  This information enables those hosting FOLIO to better manage and comply with various privacy laws and restrictions, e.g. GDPR.
+
+It's important to note that PD is not limited to that which can be used to identify a person on it's own (e.g. Social security number), but also data used in conjunction with other data to identify a person (e.g. date of birth + city + gender, or dynamic IP address + date + time), and any information about such a person (e.g. has loaned 5 items, 2 of which are overdue).
```
Comment (dltj) [inline PERSONAL_DATA_DISCLOSURE.md:5]: Good summary of the issues in _[The difference between PII and Personal Data](https://techgdpr.com/blog/difference-between-pii-and-personal-data/)_ article from TechGDPR.

**Review by dltj — APPROVED** [review]
Thanks, James.

**Review by inkuss — APPROVED** [review]
I approve the changes.

**Pull request merged**